src/Laserfiche.Api.Client.Core.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>.NET client library that enables an application to support the OAuth 2.0 Client Credentials flow.</Description>
+    <Description>.NET implementation of various foundational APIs for Laserfiche, including authorization APIs such as OAuth 2.0 flows for secure and easy access to Laserfiche APIs.</Description>
     <PackageId>Laserfiche.Api.Client.Core</PackageId>
     <Authors>Laserfiche</Authors>
     <Company>Laserfiche</Company>
-    <Version>1.0.0</Version>
+    <Version>1.0.0-beta.1</Version>
```

Comment:
are we going to need to update this version every time we release a package? I thought we were going to add the revision number to the package version in the publish to nuget pipeline